campaign_info.toml

```diff
@@ -6,24 +6,24 @@
 [desktop]
 name = "Desktop"
 icon = "desktop"
-campaign = "C24_WMDE_Desktop_DE_03"
-description = "Based on CTRL of C24_WMDE_Desktop_DE_02"
-campaign_tracking = "03-ba-240808"
+campaign = "C24_WMDE_Desktop_DE_04"
+description = "Based on C24_WMDE_Desktop_DE_04_ctrl, var: leads to different fundraising donation form (different address completion)"
```

Comment:
It is based on `C24_WMDE_Desktop_DE_03_ctrl`! (The typo is still not fixed for me 🙈)

Reply:
So sorry, I thought you meant an underscore, did not see the wrong number 🤦‍♂️